kuma/wiki/jinja2/wiki/ckeditor_config.js

```diff
@@ -151,9 +151,8 @@
         { name: 'Warning Box', element: 'div', attributes: { 'class': 'blockIndicator warning', 'role': 'region', 'aria-label': gettext('Important note') }, type: 'wrap' },
         { name: 'Two Columns', element: 'div', attributes: { 'class': 'twocolumns' }, type: 'wrap' },
         { name: 'Three Columns', element: 'div', attributes: { 'class': 'threecolumns' }, type: 'wrap' },
-        { name: 'Article Summary', element: 'p', attributes: { 'class': 'summary' } },
         { name: 'Syntax Box', element: 'pre', attributes: { 'class': 'syntaxbox' } },
-        { name: 'SEO Summary', element: 'span', attributes: { 'class': 'seoSummary' } },
+        { name: 'Summary', element: 'span', attributes: { 'class': 'seoSummary' } },
```

Comment:
For consistency, and I know this was not introduced here, can we change this to:

```suggestion
        { name: 'Summary', element: 'span', attributes: { 'class': 'seo-summary' } },
```

and then also update the class here https://github.com/mozilla/kuma/blob/master/kuma/static/styles/wiki-wysiwyg.scss#L28

@a2sheppy Thoughts?